<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="a_lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>801</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>상위폴더를 지정하시오</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>변역할 언어</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>확장자를 지정하시오</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>상품이름을 지정하시오</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>아이템의 url을 지정하시오</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>병합된 pdf 이름을 적으시오</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="q_lineEdit_1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="q_lineEdit_6"/>
      </item>
      <item>
       <widget class="QLineEdit" name="q_lineEdit_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="q_lineEdit_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="q_lineEdit_5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="q_lineEdit_4"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="reset_pushButton_1">
        <property name="text">
         <string>리셋</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_pushButton_6">
        <property name="text">
         <string>리셋</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_pushButton_2">
        <property name="text">
         <string>리셋</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_pushButton_3">
        <property name="text">
         <string>리셋</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_pushButton_5">
        <property name="text">
         <string>리셋</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_pushButton_4">
        <property name="text">
         <string>리셋</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>240</y>
     <width>160</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>이미지를 pdf로</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>만들어진 pdf를 병합</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string>이미지를 번역후 텍스트로</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reset_pushButton">
      <property name="text">
       <string>이름들과 갯수 리셋</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지크롤링</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>한번에</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 리사이즈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>병합된 pdf를 img로</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/Users/Administrator/Desktop/ugot/combinepage/ 에 들어가 있어야되요</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
